backend/src/main/java/quarano/department/web/QuestionnaireDto.java

```diff
@@ -1,12 +1,5 @@
 package quarano.department.web;
 
-import lombok.Data;
-import lombok.Setter;
-import quarano.core.validation.Textual;
-import quarano.department.Questionnaire;
-import quarano.reference.Symptom;
-import quarano.reference.SymptomRepository;
-
```

Comment:
It seems you configured a wrong import order in your IDE.
https://quarano.de/internal-test/api/docs/index.html#_developing_the_backend
Chapter 8.1.1.

Reply:
there is an import order config file for eclipse formatter -> i will commit the imports reorganized according to its autoformat in a moment.

Reply:
Unfortunately this file does not contain all the information to be set.